### PR TITLE
Set app default language as language of the user browser settings

### LINKED ISSUE #657: [FEAT] User can go directly to their language with URL path, ie say /nederlands or /espanol, etc.
# ⚠️ IMPORTANT: Please fill out this template to give us as much information as possible to consider/implement the feature.

### Prerequisites
- [x] check this box if you have completed the following:
    - Reviewed the [contributing guidelines](https://github.com/COVID-19-electronic-health-system/.github/blob/master/CONTRIBUTING.md) and [support](https://github.com/COVID-19-electronic-health-system/.github/blob/master/SUPPORT.md) files
    - Reviewed the README file for the repository you are working in
    - Searched for relevant instructions [on our Discord server](https://discord.gg/pPERUuv)
    - Searched the issues of the repository you are working in to make sure one was not already filed

### Summary

Based on user feedback from doctor in Belgium
Problem: “Which url brings me to the version where Dutch is integrated (as I use the beta link from the website I still do not see Dutch as a language).”

Action: Create React Router path (and if necessary backend endpoint to the translated version of site) in the user's native language. That way the user doesn't have to know English to navigate to the choose a language section.

For example:
Dutch:
coronatrackerbeta.com/nederlands 

Spanish:
coronatrackerbeta.com/español and coronatrackerbeta.com/espanol (in case they can't get ñ)

Corner case to consider:
For non-Latin languages like Chinese or Arabic, should we have the endpoint in their native language and English version? How do speakers access English sites in their own languages?
@AdhamAH @salvolpe @njordhov 

How does Facebook or large multilingual site do this?

### Motivation
**Why are we doing this?**
We want non-English speakers to be able to use our application easier, by letting them visit the app in their native language directly from the URL

**What use cases does it support? **
Easier user onboarding at the beginning of the app

**What is the expected outcome?**
users can access our site without having the possible roadblock of not knowing English.

### Possible Alternatives

Assume that most speakers with a cellphone know enough basic English to click on the button to choose their language. Or they will experiment and click on the button and see their language. 

### Additional Context

DocMusher is a doctor in Belgium who fits one of the user cohorts that we want to interview and gain good insights from. As an external set of eyes with domain experience dealing with COVID-19 patients, we are able to learn what is useful on the field.

### PR DESCRIPTION
**Pull request by pavel-ilin**
# ⚠️ __IMPORTANT: Please do not create a Pull Request without creating an Issue first.__

All changes need to be discussed before proceeding. Failure to do so may result in the pull request being rejected.

Before submitting a pull request, please be sure to review:
- Our Code of Conduct: https://github.com/COVID-19-electronic-health-system/.github/blob/master/CODE_OF_CONDUCT.md
- Our Contributing Guidelines: https://github.com/COVID-19-electronic-health-system/.github/blob/master/CONTRIBUTING.md
- Our Support documentation: https://github.com/COVID-19-electronic-health-system/.github/blob/master/SUPPORT.md

-----

## Please include the issue number the pull request fixes by replacing YOUR-ISSUE-HERE in the text below.

Fixes #657

## Summary

Following @njordhov advice, default language of the app set by checking web browser settings. 

## Details

Add default language of the app in `Translation` component with using `window.navigator.language`.

If user browser language is not found in the backend, by default English language will be used.  

## Test Plan (required)

yarn start
set default language in your browser
open login page

tested in Chrome and Opera browsers.

## Final Checklist

- [x] For CoronaTracker, did you bump the version in `package.json`?
    - Update the second decimal for a major change
    - Update the third decimal for a minor change
    - Numbers can go past 9, e.g. `1.0.9` => `1.0.10`
    - For more info, read about [Semantic Versioning](https://semver.org/)
- [x] Did you add any new tests as necessary?
- [x] Is your PR rebased off the most current master?
- [x] Have you squashed all commits? _(can be done at merge)_
- [x] Did you use `yarn` not `npm`? _(important!)_
- [x] Did you use Material-UI wherever possible?
- [x] Did you run `yarn lint` on the code?
- [x] Did you run all of your most recent changes locally to make sure everything is working?

### CODE REVIEW

File: client/src/components/Translations.jsx [hunk 1]
```diff
@@ -17,13 +17,20 @@ const useStyles = makeStyles(theme => ({
 export default function TranslationsMenu() {
   const classes = useStyles();
   const { i18n, t } = useTranslation();
+  const [loaded, setLoaded] = useState(false);
+
+  const setLanguage = () => {
+    setLoaded(true)
+    i18n.changeLanguage(window.navigator.language.slice(0, 2));
```
Comment (SomeMoosery) [inline client/src/components/Translations.jsx:24]: Is every language / country code 2 digits?

File: client/src/components/Translations.jsx [hunk 2]
```diff
@@ -17,13 +17,20 @@ const useStyles = makeStyles(theme => ({
...
+  }
 
   const handleClick = event => {
     i18n.changeLanguage(event.target.value);
   };
 
   return (
     <div>
+      {loaded ? null : setLanguage()}
```
Comment (SomeMoosery) [inline client/src/components/Translations.jsx:33]: Couldn't we just call `setLanguage()` in a `useEffect()` hook?

File: client/src/components/Translations.jsx (hunk 1 above)
Comment (pavel-ilin) [reply client/src/components/Translations.jsx:24]: I add Arabic, Chinese and Russian to my browser and it show that language codes consistent.  It's or `en` or `en-US` format.

File: client/src/components/Translations.jsx (hunk 2 above)
Comment (pavel-ilin) [reply client/src/components/Translations.jsx]: Done

File: client/src/components/Translations.jsx (hunk 1 above)
Comment (SomeMoosery) [reply client/src/components/Translations.jsx]: I'd say that's good for now then 👍

**Pull request merged**